imap_processing/cdf/config/imap_swe_l2_variable_attrs.yaml

```
@@ -107,7 +156,26 @@ phase_space_density_spin_sector:
   DISPLAY_TYPE: spectrogram
   FIELDNAM: Phase Space Density
   FILLVAL: -1.0000000E+31
-  UNITS: " "
+  UNITS: s^3/ (cm^6 * ster)
+  FORMAT: E14.7
+  VALIDMAX: 0.000015514
+  VALIDMIN: 0
+  VAR_TYPE: data
+
+phase_space_density:
+  CATDESC: Phase space density organized by energy(eV) and spin angle and CEM angle
```

Same for flux below.
```suggestion
  CATDESC: Phase space density organized by energy (eV), spin angle, and CEM angle
```